libs/common/include/helpers/win32_nanosleep.h

```diff
@@ -5,7 +5,7 @@
 #pragma once
 
 #include <ctime> // IWYU pragma: exports
-#ifdef _WIN32
+#if defined _WIN32 && !defined __MINGW32__
```

Comment:
a mingw compiler for windows should define _WIN32, if not, the compiler is broken

Reply:
ah forget this. its "and not mingw"... read twice, write once ...